2015-07-06  Rainer Orth  <ro@CeBiTec.Uni-Bielefeld.DE>

	* read-md.c (decimal_string): Rename to ...
	(md_decimal_string): ... this.
	(handle_enum): Reflect this.

# HG changeset patch
# Parent 8d51390422669c8929e6b44770ca6716188d001a
Rename read-md.c:decimal_string

diff --git a/gcc/read-md.c b/gcc/read-md.c
--- a/gcc/read-md.c
+++ b/gcc/read-md.c
@@ -779,7 +779,7 @@ traverse_md_constants (htab_trav callbac
 /* Return a malloc()ed decimal string that represents number NUMBER.  */
 
 static char *
-decimal_string (int number)
+md_decimal_string (int number)
 {
   /* A safe overestimate.  +1 for sign, +1 for null terminator.  */
   char buffer[sizeof (int) * CHAR_BIT + 1 + 1];
@@ -851,7 +851,7 @@ handle_enum (int lineno, bool md_p)
 	  ev->name = value_name;
 	}
       ev->def = add_constant (md_constants, value_name,
-			      decimal_string (def->num_values), def);
+			      md_decimal_string (def->num_values), def);
 
       *def->tail_ptr = ev;
       def->tail_ptr = &ev->next;
